terratest/terraform_test.go

```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-// An example of how to test the simple Terraform module in examples/terraform-basic-example using Terratest.
+// An examples of how to test the simple Terraform module in examples/terraform-basic-examples using Terratest.
```

Comment:
// An example of how to test the simple Terraform module in examples/complete using Terratest.